pkg/transform/secrets/secrets.go

```diff
@@ -42,11 +42,32 @@ type MetaData struct {
 	Namespace string `yaml:"namespace"`
 }
 
+// SecretType is an enumerator for secret types
+type SecretType int
+
+const (
+	// KeystoneSecretType - keystone type for Secret
+	KeystoneSecretType = iota
+	// HtpasswdSecretType - htpasswd type for Secret
+	HtpasswdSecretType
+	// LiteralSecretType - literal type for Secret
+	LiteralSecretType
+	// BasicauthSecretType - basicauth type for Secret
+	BasicauthSecretType
```

Comment:
Sorry I didn't catch this before you pushed that initial change, but `Basicauth` here should be `BasicAuth` with a capital A to maintain consistency with the rest of the project. Otherwise, looks good! :+1:

Reply:
Sorry, my bad :+1: